functions/process-xml-to-json/1.0/function.json

```diff
@@ -0,0 +1,32 @@
+{
+  "description": "Description",
+  "label": "Process XML to JSON",
+  "category": "Misc",
+  "icon": {
```

Comment:
Is this a correct icon for this function?

Reply:
See _chore: update naming and description_